lib/src/keyboard.dart

```diff
@@ -81,7 +82,40 @@ extension KeyboardInput on WidgetTester {
 
   Future<void> pressEnter() async {
     await sendKeyEvent(LogicalKeyboardKey.enter, platform: _keyEventPlatform);
```

Comment:
Is there a reason that you removed this `pumpAndSettle()`?

Reply:
I removed by mistake.